seedlet-blocks/block-template-parts/header.html

```diff
@@ -11,6 +11,8 @@
 <!-- wp:navigation {"itemsJustification":"center","isResponsive":true,"__unstableLocation":"primary"} -->
 <!-- /wp:navigation -->
 
+<!-- wp:navigation {"orientation":"horizontal","itemsJustification":"center","isResponsive":true,"__unstableLocation":"social"} --><!-- /wp:navigation -->
```

Comment:
For Seedlet, let's use a default icon color of black (the foreground color) to match the social menu color in the classic version of the theme.